.github/workflows/codeql-analysis.yaml

```diff
@@ -0,0 +1,35 @@
+name: CodeQL
+
```

Comment:
the same for workflow

Reply:
https://github.com/percona/percona-postgresql-operator/pull/379/commits/bc785d590629a5f3d85ee15b6dd5a07deb003165